src/components/Input/DateTimeInput/index.tsx

```diff
@@ -25,8 +27,28 @@ const Input = styled(StyledInput)`
 const StyledDay = styled.span`
   font-family: Montserrat;
   font-size: 11px;
-  color: ${props => props.theme.text5};
+  color: ${props => props.theme.text2};
 `
+const StyledDatePicker = styled(Datepicker)`
+  ::placeholder {
+    color: ${props => props.theme.text2};
+  }
+`
+const StyledCalenderIcon = styled(CalenderIcon)`
+  position: absolute;
+  top: 7px;
+  right: 8px;
+`
+
+const CustomInput = React.forwardRef(props => {
```

Comment:
Should this be a CalendarInput? It doesn't seem to be a customisable input since it has a calendar icon